### DISCUSSION TITLE
Indicator for available DetailRowExpression

### DISCUSSION THREAD

**Discussion by MarioMue13** [TabularEditor/TabularEditor3#1125] [Ideas]
I would appreciate a visual marker when a measure has a defined _DetailRowExpression_. (like a small R in the Measure Icon).

Additionally:

- better distinction in Expression Editor ( _Expression_ vs. _Detail Rows Expression_) - visually - to prevent confusion
- When _Detail Rows Expression_ is visible for Measure A and I select Measure B - the window sets back to _Expression_. I would prefer it to stay on the prior selected  _Detail Rows Expression_ view.

**Comment by mlonsk** [comment]
Hi @MarioMue13 

Thank you for the excellent suggestions, we will take a look at them

**Comment by MarioMue13** [comment]
@mlonsk  additional thought: Splittscreen in the Expression Editor - to see them side by side

**Comment by otykier** [comment]
@MarioMue13 are you aware that you can use DAX scripts to view and edit all expression on a measure (including KPI expressions, if assigned):

<img width="802" alt="image" src="https://user-images.githubusercontent.com/8976200/216310183-5e26ded8-90b4-4198-8e23-2e65ef8edbd7.png">

**Comment by MarioMue13** [comment]
@otykier yes - that would work for my "splitscreen" request

**Comment by mlonsk** [comment]
Hi @MarioMue13 
Just a quick update to your two ideas here.
We do wish to provide a way to show that a detail row expression is present, but we have to be careful how much we put onto the icons in the TOM explorer has it will get to crowded.
Would a solution like we do with KPIs be a way forward?
Here the measure has a fold out button where the KPI can be changed, and we could then do something similar to the Detail Row Expression. 
![image](https://github.com/TabularEditor/TabularEditor3/assets/11939114/1f7e32b3-a4c4-42e6-8c88-d1410cb5a17f)
To complicate the picture here, tables can also have a Detail Row Expression so we need to find a solution for them to. 
As said it is something we see value in having, but I have no timeline yet.

To the second point, this is not something we will implement. Most users would expect the behavior that is currently there and by scripting multiple measure you would be able to see the detail row expression of all.
